### DISCUSSION TITLE
Cannot configure permissions for the status field in v9

### DISCUSSION THREAD

**Discussion by cdwmhcc** [directus/directus#4709] [Archive]
In v8, I can easily configure the permissions of the `status` field. For example, articles created by users are in `draft` status by default, and will be displayed in `published` status after I pass the review. In v9, I did not find where to configure.

Did I understand it wrong? Please correct me.Thank you!

**Comment by philleepflorence** [comment]
Hi @cdwmhcc ... That has moved to [Permissions](https://docs.directus.io/concepts/permissions/) under Roles - See the [Directus Guide to Permissions](https://docs.directus.io/guides/permissions/).

**Reply by cdwmhcc** [reply]
Wow, I see,Thank you!
If you need more refined permission configuration, you need to cooperate with Filter Rules.

**Reply by cdwmhcc** [reply]
Compared with v8, this configuration will be more reasonable and free!👍👍👍
